### PR TITLE
[Term Entry] PyTorch Tensor Operations: .is_tensor()

### LINKED ISSUE #6068: [Term Entry] PyTorch Tensor Operations: .is_tensor()
### Reason/inspiration (optional)

We would like a new entry on the `.is_tensor()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [x] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Sriparno08**
### Description

Added a new term entry on `.is_tensor()` in PyTorch.

### Issue Solved

Closes #6068 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+In PyTorch, the **`.is_tensor()`** function checks if the given object is a PyTorch [tensor](https://www.codecademy.com/resources/docs/pytorch/tensors) and returns `True` if it is. This function is part of the `torch` module.
```
Comment (PragatiVerma18) [inline content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md:17]: This can be better written as:

```suggestion
In PyTorch, the **`.is_tensor()`** function checks if the given object is a PyTorch [tensor](https://www.codecademy.com/resources/docs/pytorch/tensors). It returns `True` if the object is a tensor; otherwise, it returns `False`. This function is part of the `torch` module.
```

File: content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Syntax
+
+```pseudo
+torch.is_tensor(obj, /)
```
Comment (PragatiVerma18) [inline content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md:22]:
```suggestion
torch.is_tensor(obj)
```

File: content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+- `obj`: The input object to be tested.
```
Comment (PragatiVerma18) [inline content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md:25]:
```suggestion
- `obj`: The object to check whether it is a PyTorch tensor.
```

File: content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import torch
+
+# Define an object
+obj = torch.tensor([11, 22, 33, 44, 55])
+
+# Check if the given object is a PyTorch tensor
+res = torch.is_tensor(obj)
+
+# Print the result
+print(res)
```
Comment (PragatiVerma18) [inline content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md:41]: We should show both the cases -

```suggestion
import torch

# Define objects
tensor_obj = torch.tensor([11, 22, 33, 44, 55])  # Tensor object
list_obj = [11, 22, 33, 44, 55]  # Non-tensor object

# Check if the objects are PyTorch tensors
print(torch.is_tensor(tensor_obj))  # True
print(torch.is_tensor(list_obj))    # False
```

File: content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+The above code produces the following output:
+
+```shell
+True
```
Comment (PragatiVerma18) [inline content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md:47]: The output will change based on above code changes:

```suggestion
True
False
```

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]

**Review by PragatiVerma18 — APPROVED** [review]
Hey @Sriparno08, thanks for contributing to docs. This PR looks good to be merged! 🚀
